lib/PlexAPI.js

```diff
@@ -117,7 +117,7 @@ module.exports = class PlexAPI extends EventEmitter {
           this.homey.log('[PlexAPI]', `Testing URL ${connection.uri}...`);
 
           const success = await PlexUtil.fetch({
-            url: `${connection.uri}/system`,
+            url: `${connection.uri}/identity`,
```

Comment:
What didn't work, and why does this new URL work, exactly?

Reply:
I get 404 response on /system. I find no references to this endpoint in the documentation or any other api-clients. Is this a feature on your specific device perhaps? I find /identity to be working both on local ip and remotly. I have the Plex server running on a Nvidia Shield TV Pro (2019). And this fixed the app for me.

Note: /identity can be called without a plex-token. But is lightweight and good for ping test.